cpp/models/glct_secir/parameters.h

```diff
@@ -0,0 +1,680 @@
+* limitations under the License.
+*/
+
+#ifndef MIO_GLCT_SECIR_PARAMS_H
+#define MIO_GLCT_SECIR_PARAMS_H
+
+#include "memilio/config.h"
+#include "memilio/utils/parameter_set.h"
+#include "memilio/math/eigen.h"
+#include "memilio/math/floating_point.h"
+#include "memilio/epidemiology/uncertain_matrix.h"
+#include "memilio/utils/logging.h"
+
+namespace mio
+{
+namespace glsecir
+{
+
+/***********************************************
+* Define Parameters of the GLCT-SECIHURD model *
+***********************************************/
+
+/// @brief Vector with the probability to start in any of the subcompartments of the Exposed compartment.
+struct StartingProbabilitiesExposed {
+    using Type = Eigen::VectorXd;
+    /** 
+     * @brief Default parameters can be used to get an Erlang distributed stay time in the Exposed compartment.
+     * @param[in] numExposed Number of subcompartments of the Exposed compartment.
+     */
+    static Type get_default(size_t numExposed)
+    {
+        Eigen::VectorXd def = Eigen::VectorXd::Zero(numExposed);
+        def[0]              = 1.;
+        return def;
+    }
+    static std::string name()
+    {
+        return "StartingProbabilitiesExposed";
+    }
+};
+
+/// @brief Transition Matrix of the Exposed compartment.
+struct TransitionMatrixExposedToInfectedNoSymptoms {
+    using Type = Eigen::MatrixXd;
+    /**
+     * @brief Default parameters can be used to get an Erlang distributed stay time in the Exposed compartment.
+     * @param[in] numExposed Number of subcompartments of the Exposed compartment.
+     * @param[in] timeExposed Average time spent in Exposed compartment in day unit.
+     */
+    static Type get_default(size_t numExposed, ScalarType timeExposed = 2.)
+    {
+        Eigen::MatrixXd def = Eigen::VectorXd::Constant(numExposed, -(ScalarType)numExposed / timeExposed).asDiagonal();
+        def.diagonal(1).setConstant((ScalarType)numExposed / timeExposed);
+        return def;
+    }
+    static std::string name()
+    {
+        return "TransitionMatrixExposedToInfectedNoSymptoms";
+    }
+};
+
+/// @brief Vector with the probability to start in any of the subcompartments of the InfectedNoSymptoms compartment.
+struct StartingProbabilitiesInfectedNoSymptoms {
+    using Type = Eigen::VectorXd;
+    /**
+     * @brief Default parameters can be used to get an Erlang distributed stay time in InfectedNoSymptoms compartment.
+     * @param[in] numInfectedNoSymptoms Number of subcompartments of the InfectedNoSymptoms compartment.
+     */
+    static Type get_default(size_t numInfectedNoSymptoms)
+    {
+        Eigen::VectorXd def = Eigen::VectorXd::Zero(numInfectedNoSymptoms);
+        def[0]              = 1.;
+        return def;
+    }
+    static std::string name()
+    {
+        return "StartingProbabilitiesInfectedNoSymptoms";
+    }
+};
+
+/**
+ * @brief Transition Matrix of the Phase-type distribution describing the stay time in the InfectedNoSymptoms 
```

Comment:
```suggestion
 * @brief Transition matrix of the phase-type distribution describing the stay time in the InfectedNoSymptoms 
```
Also for other compartments.